docs/openstack-quota.md

```diff
@@ -0,0 +1,37 @@
+# OpenStack Quotas
+
+To read more about Openstack quotas please visit the [upstream docs](https://docs.openstack.org/nova/rocky/admin/quotas.html).
+
+#### Viewing Your Quota
+
+You can view your quotas via Skyline or using the OpenStack CLI client.  The Skyline home page is the easiest way to view your quota limits and utilization in one place.
+![SkylineHome](./assets/images/SkylineHomeQuota.png)
+
+You can also view quota information using the OpenStack CLI, however, you'll need to use a couple of commands.<BR>
+Use the quota option to list all of your quota limits.
+
+```shell
+openstack quota show
+```
+
+![quota show](./assets/images/quota_show.png)
+
+The limits option returns quota limits along with utilization information for most services.
+
+```shell
+openstack limits show --absolute
+```
+
+![limits show](./assets/images/limits_show.png)
+
+The limits option doesn't include storage volumes, so you would also need to run the volume option.

```

Comment:
The limits_show.png displays both the maxTotalVolumes and totalVolumesUsed values, so I think openstack volume list output is not necessary.

Reply:
The show volume portion and the image file have been removed.